docs/content/_navigation.json

```diff
@@ -144,7 +144,8 @@
         ]
       },
       {
-        "title": "Schedules, Sensors, & Partitions",
+        "title": "Schedules & Sensors",
+        "path": "/concepts/partitions-schedules-sensors/schedules",
```

Comment:
We should probably also split the directories in either your upstream PR or this PR, probs the other one to minimize conflicts.

Reply:
I agree we need to do this eventually, but I recommend leaving it alone for now. Moving directories is a pretty involved process at the moment. The IA of the docs is likely to change fairly drastically in the next few months as we release the tutorial and new guides. I'm don't know that many people would clock this if we leave it, but we can re-assess if it does cause issues.

Reply:
👍